### ISSUE TITLE
How to  control  tab page num?

### ISSUE THREAD

**Issue by nfq6612** [react-native-community/react-native-tab-view#415]
It's  too slow to render all the pages now. How to render only 3 page （Current page,Left page ,right page）

**Comment by aynulhabib** [comment]
Hey there, this should do the trick: 

```
renderScene = ({ route }) => {
  if (Math.abs(this.state.index - this.state.routes.indexOf(route)) > 1) {
    return null;
  }

  return <MySceneComponent route={route} />;
};
```

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
